README.md

```diff
@@ -56,6 +56,36 @@ of the format
 to an Elasticsearch cluster at `http://localhost:9200` to an index called
 `test_data`.
 
+### Docker and Docker Compose
+
+Requires [Docker](https://docs.docker.com/get-docker/) for running the app and [Docker Compose](https://docs.docker.com/compose/install/) for running a single ElasticSearch domain with two nodes (es1 and es2).
+
+1. Set the maximum virutal memory of your machine to `262144` otherwise the ElasticSearch instances will crash, [see the docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html)
+    ```bash
+    $ sudo sysctl -w vm.max_map_count=262144
+    ```
+1. Clone this repository
+    ```bash
+    $ git clone https://github.com/oliver006/elasticsearch-test-data.git
+    $ cd elasticsearch-test-data
+    ```
+1. Run the ElasticSearch stack
+    ```bash
+    $ docker-compose up --detached
+    ```
+1. Run the app and inject random data to the ES stack
+    ```bash
+    $ docker run --rm -it --network host unfor19/es-test-data  \
```

Comment:
Can you change this to `oliver006/es-test-data` ? I'll add a commit that'll build and upload this to docker registry once we got this puppy merged.

Reply:
done